Makefile

```diff
@@ -132,6 +132,10 @@ local-unit-test: $(ENVTEST) ## Run tests.
 integration-test: $(ENVTEST) ## Run tests.
 	CGO_ENABLED=1 KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./test/integration/... -coverpkg=./...  -race -coverprofile=it-coverage.xml -v
 
+.PHONY: scheduler-integration-test
```

Comment:
can we make it as a part of "test" target? it won't be missed when others run make test command to validate the changes.